http-client/src/test/java/io/airlift/http/client/jetty/TestJettyHttpsClient.java

```diff
@@ -75,7 +76,7 @@ public void tearDownHttpClient()
         }
     }
 
-    @Test(expectedExceptions = {SSLHandshakeException.class, EOFException.class})
+    @Test(expectedExceptions = {SSLHandshakeException.class, EOFException.class, ClosedChannelException.class})
```

Comment:
@johngmyers ya, I asked the Jetty folks about that. They said we shouldn't rely on the exception type in this case